cli/coverage_cmd.go

```diff
@@ -70,18 +73,32 @@ func (c *CoverageCmd) Execute(args []string) error {
 	return nil
 }
 
-var codeExts = []string{".go", ".java", ".py", ".rb", ".cpp", ".ts", ".cs", ".js", ".php", ".m"} // codeExt lists all file extensions that indicate a code file we want to cover
-var codeExts_ = make(map[string]struct{})
+var langToExts = map[string][]string{
+	"Go":          []string{".go"},
+	"Java":        []string{".java"},
+	"Python":      []string{".py"},
+	"Ruby":        []string{".rb"},
+	"C++":         []string{".cpp"},
+	"TypeScript":  []string{".ts"},
+	"C#":          []string{".cs"},
+	"JavaScript":  []string{".js"},
+	"PHP":         []string{".php"},
+	"Objective-C": []string{".m"},
+}
+var extToLang map[string]string
```

Comment:
Can we use the existing filelang pkg to determine this? Should we?

Sent from my iPhone

> On Apr 5, 2016, at 21:54, Alex Saveliev <notifications@github.com> wrote:
> 
> In cli/coverage_cmd.go:
> 
> > @@ -70,18 +73,32 @@ func (c *CoverageCmd) Execute(args []string) error {
> >  	return nil
> >  }
> >  
> > -var codeExts = []string{".go", ".java", ".py", ".rb", ".cpp", ".ts", ".cs", ".js", ".php", ".m"} // codeExt lists all file extensions that indicate a code file we want to cover
> > -var codeExts_ = make(map[string]struct{})
> > +var langToExts = map[string][]string{
> > +	"Go":          []string{".go"},
> > +	"Java":        []string{".java"},
> > +	"Python":      []string{".py"},
> > +	"Ruby":        []string{".rb"},
> > +	"C++":         []string{".cpp"},
> > +	"TypeScript":  []string{".ts"},
> > +	"C#":          []string{".cs"},
> > +	"JavaScript":  []string{".js"},
> > +	"PHP":         []string{".php"},
> > +	"Objective-C": []string{".m"},
> > +}
> > +var extToLang map[string]string
> What if there are multiple file extensions denoting the same language (example: .c++, .cxx, .cpp, .cc)?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>